### PR TITLE
Plant illustration

### PR DESCRIPTION
**Pull request by labichn**
### Description
This PR adds the plant illustration to the component library.

### Testing Instructions
Look at the gorgeous plant!

### CODE REVIEW

File: components/atoms/illustrations/PlantCircleIllustration/PlantCircleIllustration.js [hunk 1]
```diff
@@ -0,0 +1,2771 @@
...
+            <rect height="3.63" rx="1.5" width="29.5" x="36.65" y="66.75" />
+            <rect height="3.63" rx="1.5" width="29.5" x="36.65" y="70.38" />
+            <rect height="3.63" rx="1.5" width="29.5" x="36.65" y="74.01" />
+            <rect height="3.63" rx="1.5" width="29.5" x="36.65" y="77.63" />
+            <rect height="3.63" rx="1.5" width="29.5" x="36.65" y="81.26" />
+            <rect height="3.63" rx="1.5" width="29.5" x="36.65" y="84.89" />
+            <rect height="3.63" rx="1.5" width="29.5" x="36.65" y="88.51" />
+            <rect height="3.63" rx="1.5" width="29.5" x="36.65" y="92.14" />
+            <rect height="3.63" rx="1.5" width="29.5" x="36.65" y="95.77" />
+            <rect height="3.63" rx="1.5" width
```
Comment (JamesIves) [inline components/atoms/illustrations/PlantCircleIllustration/PlantCircleIllustration.js]: This should be updated to `plant-circle-illustration`

File: components/atoms/illustrations/PlantCircleIllustration/PlantCircleIllustration.spec.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  const component = shallow(<PlantCircleIllustration draft fill="white" />);
+
+  t.equals(
+    component.find('svg').length,
+    1,
+    'Should load the PlantIllustration component.'
+  );
+
+  t.end();
+});
```
Comment (CalMlynarczyk) [inline components/atoms/illustrations/PlantCircleIllustration/PlantCircleIllustration.spec.js]: Can we add some sort of differentiating checks between these three to ensure that the correct SVG is getting returned and not just that any SVG is returned?

Comment (JamesIves) [reply components/atoms/illustrations/PlantCircleIllustration/PlantCircleIllustration.spec.js]: Make the id `${id}-draft`

File: components/atoms/illustrations/PlantCircleIllustration/PlantCircleIllustration.js (hunk 1 above)
Comment (JamesIves) [reply components/atoms/illustrations/PlantCircleIllustration/PlantCircleIllustration.js]: This is meant to flag the `defaultProp` not the `rect` tag?

Comment (labichn) [reply components/atoms/illustrations/PlantCircleIllustration/PlantCircleIllustration.js]: Agreed, and fixed in 55b2836.

File: components/atoms/illustrations/PlantCircleIllustration/PlantCircleIllustration.spec.js (hunk 2 above)
Comment (labichn) [reply components/atoms/illustrations/PlantCircleIllustration/PlantCircleIllustration.spec.js]: Agreed, and fixed in 55b2836.

**Pull request merged**